frontend/src/routes/accounts/+page.svelte

```diff
@@ -39,6 +39,7 @@
 					Co-owned by {coOwners.map((owner) => accountName(owner)).join(', ')}
 				</p>
 			{/if}
+			{#if serverState.isAdmin }
```

Comment:
This is fine for Ash but we should leave a TODO comment or issue to undo it for next time we have a bootcamp with coding